src/applications/proxy-rewrite/README.md

```diff
@@ -11,8 +11,12 @@ This works by -
 ## Local Dev
 Because we're dependent on the DOM of webpages outside of our source code, local development is somewhat tricky. However, to make this easier, you can run the watch task with a flag indicating that `proxy-rewrite` development, which will ultimately serve as a local proxy. The reason this behavior is not enabled by default is that we don't want to hit the production servers unless we have to, which is the case for TeamSite development.
 
+First, make sure there exists a `headerFooter.json` file in `vets-website/build/localhost/generated/`. This file should be generated by running a build on `content-build`.
+
+Now, start up `vets-website`:
+
 ```
-npm run watch -- --local-proxy-rewrite
+yarn run watch --env --local-proxy-rewrite
```

Comment:
```suggestion
yarn run watch --env local-proxy-rewrite
```

Reply:
@ddzz Can you shed some light on why you're including `--env` in this command? It doesn't seem strictly necessary to me.

Reply:
@ddzz Thanks for the feedback! I _believe_ that this is a slightly different situation than the examples given in the main `README` in that those examples are setting webpack environment variables. In this case, we shouldn't need that. I'm open to feedback from others, but I'd like to update the documentation as accurately as possible if we're going to update it.

This was a change that you helped with a lot, though, Darius, so I'll leave this up to you.  @allisonlu, I'll approve and we can change or update based on Darius's call.

Thanks to you both.

Reply:
Either way works, so I have no preference here.